### PR TITLE
dotCMS/core#23088 System Container shows broken images when adding

### PR DESCRIPTION
**Pull request by rjvelazco**
**What we did here**

1. Update velocity code to display images only when needed.
2. Add a card style for the `system container` content.

**Card Style**
|  Desktop |  Responsive  |
|---|---|
|  desktop | responsive |

**Behavior**

https://user-images.githubusercontent.com/72418962/196234798-df022580-54c3-4a4e-962b-0cfc69241bb9.mov

### CODE REVIEW

**Comment by rjvelazco** [comment]
Hey, @wezell, I hope you are doing great!

I talked to Jason and he came up with this card style. Can you take a look to see if you like it?

system_container

**Review by oidacra — APPROVED** [review]

File: dotCMS/src/main/webapp/WEB-INF/velocity/static/system_theme/template.vtl [hunk 1]
```diff
@@ -55,6 +55,121 @@ body {
...
+.card_title h1 {
+    overflow: hidden;
+    font-size: 40px;
+    margin: 0;
+    margin-bottom: 10px;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.badge {
```
Comment (hmoreras) [inline dotCMS/src/main/webapp/WEB-INF/velocity/static/system_theme/template.vtl:110]: add more specificity to this classes and bellow this one.

**Review by hmoreras — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/resources/com/dotmarketing/portlets/containers/business/default_container.vtl [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+            <img src="/dA/$!{contentlet.inode}" />
+        </div>
+    #end
+
+    <div class="card_body">
+        <div class="card_header">
+            <div class="card_title">
+                <h1>#if($title)$title#{else}$!contentlet.title#end</h1>
+            </div>
+            <span class="badge">#if($!{contentMap})$contentType.name()#{else}DRAFT#end</span>
```
Comment (jdotcms) [inline dotCMS/src/main/resources/com/dotmarketing/portlets/containers/business/default_container.vtl:24]: DRAFT shouldn't be i18n

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/main/resources/com/dotmarketing/portlets/containers/business/default_container.vtl [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+        </div>
+        <div class="card_footer">
+            <div class="status #if($contentlet.isLive())live#{else}working#end"></div>
```
[the remark this answers is not in the document]
Comment (rjvelazco) [reply dotCMS/src/main/resources/com/dotmarketing/portlets/containers/business/default_container.vtl:27]: In this case, `live` and `working` work as `CSS` classes.

**Review by hmoreras — APPROVED** [review]

**Pull request merged**